plugins/multilanguage.ts

```diff
@@ -150,6 +150,23 @@ export default function multilanguage(userOptions: Options) {
       }
     });
 
+    /**
+     * Preprocessor to process the Unmatched Language URL
+     *
+     * + convert unmatchedLangUrl any value to URL string value
+     */
+    site.preprocess(options.extensions, (filteredPages) => {
+      for (const page of filteredPages) {
+        const unmatchedLangPath = getUnmatchedLangPath(
+          page,
+          filteredPages,
+        );
+        page.data.unmatchedLangUrl = unmatchedLangPath
+          ? site.url(unmatchedLangPath, true)
```

Comment:
I wouldn't use `site.url()` here. It's better just store the url path.

Reply:
Sure I can, it'll make my life much easier. Still I would like to come up a case with path is needed instead of url?

Reply:
path is more flexible because allows to create relative urls easier. And `site.url()` can be used at any time if you need absolute urls (in fact, it's already used in sitemap).